android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -415,22 +431,30 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       setText(view, R.id.description, pwsResult.getDescription());
       ((ImageView) view.findViewById(R.id.icon)).setImageBitmap(
           Utils.getBitmapIcon(mPwCollection, pwsResult));
+      final String siteUrl = pwsResult.getSiteUrl();
+        if (Utils.isFavorite(pwsResult.getSiteUrl())) {
+          ((ImageView) view.findViewById(R.id.star)).setImageResource(R.drawable.ic_star_black_24dp);
```

Comment:
Can't you just get stars from here? https://developer.android.com/reference/android/R.drawable.html